Resources/doc/configuration.rst

```diff
@@ -1,6 +1,17 @@
 Configuration
 =============
 
+Camel case
+----------
+By default, the serialization is done in snake_case format. To switch to camelCase
+format, please add the following configuration parameters to jms_serializer.yaml file:
+
+.. code-block :: yaml
+    jms_serializer:
+        ...
+        property_naming:
+            id: 'jms_serializer.identical_property_naming_strategy'
```

Comment:
what you are suggesting here is wrong. as you can read "identical_property_naming_strategy" means "use the same property name as in the class", does not mean camel case.

the right config for camel case is

```yml
    jms_serializer:
        ...
        property_naming:
          separator:  ''
          lower_case: false
```

Reply:
sooo,

let me make it clear - you knew the correct solution from the very beginning, and, instead of fixing the documentation yourself, [here](https://github.com/schmittjoh/JMSSerializerBundle/issues/835) you told me to include the solution from [here](https://github.com/schmittjoh/JMSSerializerBundle/issues/734#issuecomment-791621841) and, after I did that, now you are telling me the solution was wrong.

like, dude, are you okay? also, if you think I'm a clown here to entertain you with commits, you are probably very, very wrong.